bin/gitea-push.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+. bin/common.sh
+[ "$CI" = 'true' ] && exit 1
+set -eo pipefail
+prepare_crypt
+readonly values=$(hf_values)
+readonly gitea_enabled=$(echo "$values" | yq r - 'charts.gitea.enabled')
+[ "$gitea_enabled" != "true" ] && err "Gitea is disabled" && exit 1
+
+readonly gitea_url=$(echo "$values" | yq r - 'cluster.domain')
+readonly gitea_password=$(echo "$values" | yq r - 'charts.gitea.admin.password')
+readonly gitea_user='otomi'
+readonly gitea_repo='values'
+readonly remote_name='origin'
+
+cd $ENV_DIR
+
+# Initialize as clean slate
+if [ ! -d .git ]; then
+  git init
+fi
+# Check if gitea is origin, otherwise stop
+if [ $(git config remote.$remote_name.url) ] && [ "gitea.$gitea_url" != $(git config remote.$remote_name.url | cut -d@ -f2 | cut -d/ -f1 | cut -d: -f1) ]; then
```

Comment:
I am always struggling with reading bash code that parses some strings.

Wouldn't be easier to compose expected git url string (e.g.: `git remote add $remote_name "https://$gitea_user:$gitea_password@gitea.$gitea_url/$gitea_user/$gitea_repo.git` and compare it with against existing git config ?

Reply:
I could do that, but what if `gitea.$cluster_domain` is already set as remote, but maybe with a different user/repo/signing?

Reply:
Well then, would it work if `remote` does not follow our convention ?

Reply:
If the remote is SSH for example and the keys are set, the script would still work. Eg get number of commits, if 0 commit values, else print message.

Reply:
Very rare case, but can happen.

Reply:
Why would that occur? This script is the initiator of this process. Please think about real use cases, and avoid coming up with exotic what ifs that we don't care to support.